Tests/Tests/LoginTest.cs

```diff
@@ -9,15 +9,27 @@ public class LoginTest : BasicTest
         [Test]
         public void FirstTest()
         {
-            var loginBtn = mainPage.GoToLoginPage().GoToAccountPage();
+            var btn = mainPage.GoToLoginPage().GoToAccountPage();
```

Comment:
Since `GoToAccountPage()` returns page object you sould name variable appropriatively.